ragna/core/_rag.py

```diff
@@ -191,7 +191,7 @@ async def prepare(self) -> Message:
                 "Chat is already prepared",
                 chat=self,
                 http_status_code=400,
-                detail=RagnaException.EVENT,
+                http_detail=RagnaException.EVENT,
```

Comment:
Driveby because I hit this during development and I was wondering why I don't get diagnostics back.